plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergPlugin.java

```diff
@@ -196,4 +206,19 @@ private static ConnectorFactory getConnectorFactory()
     {
         return getOnlyElement(new IcebergPlugin().getConnectorFactories());
     }
+
+    @Test
+    public void testNessieMetastore()
+    {
+        ConnectorFactory factory = getConnectorFactory();
+
+        factory.create(
+                        "test",
+                        Map.of(
+                                "iceberg.catalog.type", "nessie",
+                                "iceberg.nessie.default-warehouse-dir", "/tmp",
+                                "iceberg.nessie.uri", "http://foo:1234"),
```

Comment:
nit: Can we use a proper nessie uri i.e with some valid path ?

Reply:
I had that initially, but it was suggested to update the value to align with the other test methods in this class